feature_engine/testing/data/cleaned_data/test_chat_level.csv

```diff
@@ -148,4 +148,534 @@ I respond to that too",num_block_quote_responses,2
+0,0,thereforrnsq,causation_lexical_per_100,0.01
+0,0,origins,causation_lexical_per_100,0.01
+0,0,motivnt launchvt,causation_lexical_per_100,0.02
+0,0,makes forcev using obedienwayd,causation_lexical_per_100,0.04
+0,0,compliance obedienlpijv pick,causation_lexical_per_100,0.03
+0,0,sucked arrogant ferocwv,anger_lexical_per_100,0.03
+0,0,punishx enragvbj contemptawb,anger_lexical_per_100,0.03
+0,0,maniacxo pissmygt rebelmp,anger_lexical_per_100,0.03
+0,0,stupidncwg hellish paranoilbock warring,anger_lexical_per_100,0.04
+0,0,naga angrwmnl tantrumsv,anger_lexical_per_100,0.03
+0,0,ive I've,first_person_singular_lexical_per_100,0.02
+0,0,Id,first_person_singular_lexical_per_100,0.01
+0,0,Id myself,first_person_singular_lexical_per_100,0.02
+0,0,I'd,first_person_singular_lexical_per_100,0.01
+0,0,i,first_person_singular_lexical_per_100,0.01
+0,0,weightliftd hand driemau drily,feel_lexical_per_100,0.04
+0,0,grippfipun feelingwkaf thinnajy,feel_lexical_per_100,0.03
+0,0,skin'men,feel_lexical_per_100,0.01
+0,0,weight press hot thin silkvr,feel_lexical_per_100,0.05
+0,0,skin'h,feel_lexical_per_100,0.01
+0,0,assumbvd seemed usually,tentativeness_lexical_per_100,0.03
+0,0,option,tentativeness_lexical_per_100,0.01
+0,0,someonesyc wonders theorhe,tentativeness_lexical_per_100,0.03
+0,0,spose hardly seemed alot,tentativeness_lexical_per_100,0.04
+0,0,often appearing,tentativeness_lexical_per_100,0.02
+0,0,either except sometime,exclusive_lexical_per_100,0.03
+0,0,without,exclusive_lexical_per_100,0.01
+0,0,really but whether,exclusive_lexical_per_100,0.03
+0,0,vs,exclusive_lexical_per_100,0.01
+0,0,if,exclusive_lexical_per_100,0.01
+0,0,begin supported,verbs_lexical_per_100,0.02
+0,0,begin held who's describe how's,verbs_lexical_per_100,0.05
+0,0,takes,verbs_lexical_per_100,0.01
+0,0,happened saw mustnt brings,verbs_lexical_per_100,0.04
+0,0,slept took waited,verbs_lexical_per_100,0.03
+0,0,pals bud amigodyzg mate's girlfriendur,friends_lexical_per_100,0.05
+0,0,mates,friends_lexical_per_100,0.01
+0,0,neighborwtey bftxuv comradq,friends_lexical_per_100,0.03
+0,0,colleaguejo buddiesef,friends_lexical_per_100,0.02
+0,0,bfbfi,friends_lexical_per_100,0.01
+0,0,an a alot the,article_lexical_per_100,0.04
+0,0,the,article_lexical_per_100,0.01
+0,0,an,article_lexical_per_100,0.01
+0,0,an the,article_lexical_per_100,0.02
+0,0,an,article_lexical_per_100,0.01
+0,0,u know no,argue_lexical_per_100,0.02
+0,0,really you know I think,argue_lexical_per_100,0.03
+0,0,really I believe,argue_lexical_per_100,0.02
+0,0,cause,argue_lexical_per_100,0.01
+0,0,I know,argue_lexical_per_100,0.01
+0,0,you'd could,auxiliary_verbs_lexical_per_100,0.02
+0,0,couldnt isn't,auxiliary_verbs_lexical_per_100,0.02
+0,0,oughta had be what's,auxiliary_verbs_lexical_per_100,0.04
+0,0,be shant itll,auxiliary_verbs_lexical_per_100,0.03
+0,0,ain't couldnt,auxiliary_verbs_lexical_per_100,0.02
+0,0,avertv,cognitive_mech_lexical_per_100,0.01
+0,0,yearnqgjth,cognitive_mech_lexical_per_100,0.01

```

Comment:
Everything looks good, but I notice that most of these test cases are absolutely nonsense? I guess it doesn't matter too much what the sentences are, as long as they're just testing that the lexical words are in there, but some of these don't even seem like real English words, lol.

Can you say more about how these were generated?